library/src/main/java/com/alamkanak/weekview/WeekViewEvent.java

```diff
@@ -65,13 +65,49 @@ public WeekViewEvent(long id, String name, int startYear, int startMonth, int st
      * Initializes the event for week view.
      * @param id The id of the event.
      * @param name Name of the event.
+     * @param startYear Year when the event starts.
+     * @param startMonth Month when the event starts.
+     * @param startDay Day when the event starts.
+     * @param startHour Hour (in 24-hour format) when the event starts.
+     * @param startMinute Minute when the event starts.
+     * @param endYear Year when the event ends.
+     * @param endMonth Month when the event ends.
+     * @param endDay Day when the event ends.
+     * @param endHour Hour (in 24-hour format) when the event ends.
+     * @param endMinute Minute when the event ends.
+     */
+    @Deprecated
+    public WeekViewEvent(long id, String name, int startYear, int startMonth, int startDay, int startHour, int startMinute, int endYear, int endMonth, int endDay, int endHour, int endMinute) {
+        this.mId = Integer.toString((int) id);
+
```

Comment:
you should call the constructor with the String attribute after converting instead of copy pasting.

Reply:
The conversion should happen with `String.valueOf(id);`